### PR TITLE
Show linkout to Marketplace when there are no product metabox suggestions to display

### PR DESCRIPTION
**Pull request by haszari**
### Changes proposed in this Pull Request:
When a user has dismissed all the product metabox suggestions, or if there are none to display, the UI is a little confusing.

This PR shows a nice message and button linking to the marketplace so it's clear why it's there.

### How to test the changes in this Pull Request:
1. Clone this branch and get it running as your woocommerce extension in a test environment.
2. Edit a product, scroll down to metabox, click `Get more options` tab.
3. Dismiss all the suggestions – should see nice new UI.

Also, after dismissing all the suggestions, reload the page, and click on the tab.

### Screenshots

Screen Shot 2019-03-13 at 9 50 23 AM

### CODE REVIEW

**Review by kloon — CHANGES_REQUESTED** [review]
Not getting the same results, in my tests after dismissing all the suggestions it first showed a blank page and then removed the tab for suggestions. Then I reloaded the edit product page for the product and the tab was still missing.

I am in the right branch https://cld.wthms.co/i0DYSJ, here is a screencast after clearing cookies and the user meta

**Comment by haszari** [comment]
> I am in the right branch https://cld.wthms.co/i0DYSJ, here is a screencast after clearing cookies and the user meta 

Apologies @kloon – I forgot to push my last commit.

(Thanks for testing the fallback option of hiding everything though.)

File: includes/admin/marketplace-suggestions/templates/html-product-data-extensions.php [hunk 1]
```diff
@@ -17,4 +17,12 @@
 		WC_Marketplace_Suggestions::render_suggestions_container( 'product-edit-meta-tab-body' );
 		WC_Marketplace_Suggestions::render_suggestions_container( 'product-edit-meta-tab-footer' );
 	?>
+	<div class="marketplace-suggestions-metabox-nosuggestions-placeholder hidden">
+		<img src="https://woocommerce.com/wp-content/plugins/wccom-plugins/marketplace-suggestions/icons/get_more_options.svg" class="marketplace-suggestion-icon">
+		<div class="marketplace-suggestion-placeholder-content">
+			<h4>Enhance your products</h4>
```
Comment (timmyc) [inline includes/admin/marketplace-suggestions/templates/html-product-data-extensions.php]: These two lines need to be translated.

**Review by timmyc — CHANGES_REQUESTED** [review]
This is testing well for me - just the one comment having the two sentences passed through the translate method, and all is looking good here.

File: includes/admin/marketplace-suggestions/templates/html-product-data-extensions.php (hunk 1 above)
Comment (haszari) [reply includes/admin/marketplace-suggestions/templates/html-product-data-extensions.php]: Woops – now fixed.

**Review by timmyc — APPROVED** [review]

**Comment by timmyc** [comment]
Still testing out well for me, thanks for fixing up the translations.

File: includes/admin/marketplace-suggestions/templates/html-product-data-extensions.php [hunk 2]
```diff
@@ -17,4 +17,12 @@
 		WC_Marketplace_Suggestions::render_suggestions_container( 'product-edit-meta-tab-body' );
 		WC_Marketplace_Suggestions::render_suggestions_container( 'product-edit-meta-tab-footer' );
 	?>
+	<div class="marketplace-suggestions-metabox-nosuggestions-placeholder hidden">
+		<img src="https://woocommerce.com/wp-content/plugins/wccom-plugins/marketplace-suggestions/icons/get_more_options.svg" class="marketplace-suggestion-icon">
+		<div class="marketplace-suggestion-placeholder-content">
+			<h4><?php esc_html_e( 'Enhance your products', 'woocommerce' ); ?></h4>
+			<p><?php esc_html_e( 'Extensions can add new functionality to your product pages that make your store stand out', 'woocommerce' ); ?></p>
+		</div>
+		<a href="https://woocommerce.com/product-category/woocommerce-extensions/?utm_source=editproduct&amp;utm_campaign=marketplacesuggestions&amp;utm_medium=product" target="blank" class="button">Browse the Marketplace</a>
```
Comment (mikejolley) [inline includes/admin/marketplace-suggestions/templates/html-product-data-extensions.php]: This button is not translatable.

**Review by mikejolley — DISMISSED** [review]
There is still a string missing translation functions. There is also a merge conflict so you may need to regenerate that minified file.

**Comment by haszari** [comment]
All fixed up now @mikejolley. Have rebased onto master & recompiled the js to pick up the upstream changes (hiatus in products list) & tested this is all working correctly.

**Comment by haszari** [comment]
Has been re-rebased and minified js rebuilt to catch the upstream changes (https://github.com/woocommerce/woocommerce/pull/23025), totally 100% ready for merge :)

**Review by kloon — APPROVED** [review]
All good now, thanks!

**Pull request merged**